.github/workflows/_test.yml

```diff
@@ -75,7 +103,7 @@ jobs:
     # Always run when all 'tests' jobs have finished even if they failed
     # TODO: Replace always() with a 'at least one job succeeded' expression
     if: always()
-    needs: tests
+    needs: [lint-test, tests]
```

Comment:
Shouldn't this work as is by transitivity? If "coveralls-fin" needs "tests" and "tests" needs "lint-test", then 
"coveralls-fin" needs "lint-test" without specifying it explicitly, right?

Reply:
Yes, you are right. Just fixed it.